indico/modules/events/templates/management/_reference_list.html

```diff
@@ -0,0 +1,14 @@
+{% macro render_event_references_list(event, references) %}
```

Comment:
you don't use the event for anything - so i'd either pass the object and then use `whatever.references` or pass just the list of references